src/legacy/server/config/schema.js

```diff
@@ -245,6 +245,7 @@ export default () =>
         }),
         faviconUrl: Joi.any().default('/'),
         applicationTitle: Joi.any().default(''),
+        useExpandedMenu: Joi.boolean().default(false),
```

Comment:
I think if we want this in 2.1 we should default this to `true`

Reply:
updated in latest revision